bucket/file-optimizer.json

```diff
@@ -0,0 +1,50 @@
+{
+    "version": "14.10.2534",
+    "description": "File optimizer featuring lossless file size reduction",
+    "homepage": "https://nikkhokkho.sourceforge.io/static.php?page=FileOptimizer",
+    "license": "AGPL-3.0-or-later",
+    "url": "https://downloads.sourceforge.net/project/nikkhokkho/FileOptimizer/14.10.2534/FileOptimizerFull.7z.exe#/dl.7z",
+    "hash": "sha1:a6ba351d9f5cfbd647c64b37f2d4b26c3c2bfd4d",
+    "architecture": {
+        "32bit": {
+            "bin": [
+                [
+                    "FileOptimizer32.exe",
+                    "FileOptimizer"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "FileOptimizer32.exe",
+                    "File Optimizer"
+                ]
+            ]
+        },
+        "64bit": {
+            "bin": [
+                [
+                    "FileOptimizer64.exe",
+                    "FileOptimizer"
+                ]
+            ],
+            "shortcuts": [
+                [
+                    "FileOptimizer64.exe",
+                    "File Optimizer"
+                ]
+            ]
+        }
+    },
+    "pre_install": "'32', '64' | % { if (!(Test-Path $persist_dir\\FileOptimizer$_.ini)) { New-Item | Out-Null } }",
```

Comment:
Are you kidding me?

I hope that need for editing it as normal, readable and working PowerShell is for someone who wrote so much manifest do not have to be mentioned...

Reply:
Now it should be working properly. Really sorry for that.